### DISCUSSION TITLE
How to always render box-drawing symbols as bold?

### DISCUSSION THREAD

**Discussion by ekhodx** [wezterm/wezterm#4280] [Q&A] [answered]
![image](https://github.com/wez/wezterm/assets/49815452/08395172-b455-414e-b966-4ae262e72b1a)

Two top terminals are wezterm and two bottom are alacritty. You can see that prompt from neovim's plugin "noice" (left side) is pretty damn ugly. How do I achieve same bold symbols as in alacritty? (right side is lazygit)

I am using `Fira Code` font as you can see in config file.

UPD: `custom_block_glyphs = false` does render border symbols as bold and it looks identical as in alacritty, but now some symbols are broken, e.g. my oh-my-posh theme:
![image](https://github.com/wez/wezterm/assets/49815452/2f5053e1-28ce-46d1-927b-c32054748eec)

**Comment by wez** [comment, marked as answer]
try: https://wezfurlong.org/wezterm/config/lua/config/underline_thickness.html

**Reply by ekhodx** [reply]
Oof.. okay, it works :D. Thank you. I also had to change cursor width as this setting affects it too.
